instance/migrations/0115_auto_20190322_1036.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
Give a meaningful name to the migration (ref. https://docs.djangoproject.com/en/2.1/topics/migrations/#workflow)